main.tf

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
this `main.tf` is kinda big
I suggest to split it into more files logically grouped (i.e. `main.tf`, `gcp.tf`, `velocloud.tf`)

Reply:
Makes sense to me, it is a little long and hard to reed. I will attempt to break apart the main.tf file